### DISCUSSION TITLE
Page 1 url went wrong..

### DISCUSSION THREAD

**Discussion by ywbird** [mmistakes/minimal-mistakes#3168] [Q&A] [answered]
![paginator](https://user-images.githubusercontent.com/83404333/134853228-1cba678e-f787-4502-9c25-ffd0bb03d997.png)
When I click 1️⃣
![went wrong](https://user-images.githubusercontent.com/83404333/134854502-6e958d75-edb3-4741-a74e-c02ae3689137.png)
and go to 404 page...
Please help...

[my _config.yml](https://github.com/ywbook/blog/blob/master/_config.yml)
[my paginator.html](https://github.com/ywbook/blog/blob/master/_includes/paginator.html)
[my website](https://ywbook.github.io/blog/)

**Comment by iBug** [comment, marked as answer]
You need to replace `page:num` with `page=:num` in [`paginator.html` here][1] if you want this scheme. The theme expected you to use `page:num` out-of-the-box.

  [1]: https://github.com/ywbook/blog/blob/3204bb799960313271aba561cb28ac2a474132bd/_includes/paginator.html#L3

**Reply by ywbird** [reply]
It worked!! Thanks!
